x-pack/plugins/ml/public/application/jobs/new_job/pages/job_type/page.tsx

```diff
@@ -106,15 +106,15 @@ export const Page: FC = () => {
       icon: {
         type: 'createMultiMetricJob',
         ariaLabel: i18n.translate('xpack.ml.newJob.wizard.jobType.multiMetricAriaLabel', {
-          defaultMessage: 'Multi metric job',
+          defaultMessage: 'Multi-metric job',
         }),
       },
       title: i18n.translate('xpack.ml.newJob.wizard.jobType.multiMetricTitle', {
-        defaultMessage: 'Multi metric',
+        defaultMessage: 'Multi-metric',
```

Comment:
If you are adding the hyphen here, I think we should be hyphenating `multi-metric` across all our pages.  Looks like you should also edit here:

```
./public/application/jobs/new_job/common/job_creator/multi_metric_job_creator.ts (just used in a comment)
./public/application/jobs/new_job/common/job_creator/util/general.ts
./public/application/jobs/new_job/pages/components/pick_fields_step/components/single_metric_view/settings.tsx
./public/application/routing/routes/new_job/wizard.tsx
```

![image](https://user-images.githubusercontent.com/7405507/80962446-f8003d00-8e04-11ea-8f36-bea0f5b21f89.png)

![image](https://user-images.githubusercontent.com/7405507/80962512-149c7500-8e05-11ea-800d-be08df4bd8a3.png)